### ISSUE TITLE
Cannot Login through the console in windows

### ISSUE THREAD

**Issue by rafareyes7** [heroku/heroku#1711]
Hi, I'm trying to login in heroku , but I just can't.. I don't get any response .. just keep waiting for asking my credentials.

Rafael@rr7 MINGW64 ~/Development/RubyOnRails/chef_blog (master)
$ heroku version
DL is deprecated, please use Fiddle
heroku-toolbelt/3.41.5 (i386-mingw32) ruby/2.1.5
You have no installed plugins.

Rafael@rr7 MINGW64 ~/Development/RubyOnRails/chef_blog (master)
$ heroku login
DL is deprecated, please use Fiddle

Can you guys help me?

**Comment by dickeyxxx** [comment]
I'm not sure there is a whole lot I can do to help without an error message. The one weird thing is that it says you're running 2.1.5 but using the toolbelt it should be 1.9.3. That might mean your install isn't using the heroku provided version of ruby and possibly a version of ruby with some problems.

**Comment by rafareyes7** [comment]
Yes, I'm trying to avoid the setud install the Ruby 1.9.3 because I'm running the 2.1.5 so I've downloaded the Heroku Client (Standalone - "https://s3.amazonaws.com/assets.heroku.com/heroku-client/heroku-client.tgz"), extract the files in C:\Program Files (x86)\Heroku and add it to the PATH. following the post http://stackoverflow.com/questions/12947936/installing-heroku-client-only-and-not-the-other-things.

**Comment by dickeyxxx** [comment]
oh that's your problem, that won't work on windows

**Comment by rafareyes7** [comment]
=( so how can I make it works on Windows? Is there any solution?

**Comment by dickeyxxx** [comment]
you NEED to install the toolbelt from the windows package with 1.9.3. That doesn't mean you have to use 1.9.3 for anything else though.

**Comment by rafareyes7** [comment]
I Just did it, so I try to login and I get the same result..


Rafael@rr7 MINGW64 ~/Development/RubyOnRails/chef_blog (master)
$ heroku version
heroku/toolbelt/3.41.5 (i386-mingw32) ruby/1.9.3
You have no installed plugins.

Rafael@rr7 MINGW64 ~/Development/RubyOnRails/chef_blog (master)
$ heroku login

**Comment by dickeyxxx** [comment]
I imagine the PATH isn't setup such that it's pointing to the .bat file in your program files directory

**Comment by rafareyes7** [comment]
Here's my PATH: 
%SystemRoot%\system32;%SystemRoot%;%SystemRoot%\System32\Wbem;%SYSTEMROOT%\System32\WindowsPowerShell\v1.0\;C:\Program Files\nodejs\;C:\Program Files (x86)\Heroku\bin;C:\Program Files (x86)\git\cmd

There's something wrong that I should set up?
